crowbar_framework/app/helpers/nodes_helper.rb

```diff
@@ -16,8 +16,12 @@
 #
 
 module NodesHelper
+  def total_nodes
```

Comment:
somehow yes, I have just used split the old name as it was without thinking too much about it. l'll change it